domain/src/Item.js

```
@@ -21,9 +21,11 @@ function calcAps(item) {
 
 function calcDps(item) {
     let res = (((item.minDmg + item.maxDmg) / 2) * item.aps);
-    return (item.critChance && item.critDmg)
-        ? (res + (item.critChance / 100) * (item.critDmg / 100))
-        : res;
+    if (item.critChance && item.critDmg) {
+        if (item.critChance > 100) item.critChance = 100;
```

and don't silently ignore my comments :angry: i told you about tests for calcDps and you neither added them nor responded with anything.